submissions/Levi123/html-css-popup/index.html

```diff
@@ -0,0 +1,111 @@
+                            <div>Search</div>
+                        </li>
+                        <li class="elementPopUp">
+                            <a href="#"><img src="icons/photos.png" alt = "photos" class = "imgMenu"/></a>
+                            <div>Photots</div>
+                        </li>
+                        <li class="elementPopUp">
+                            <a href="#"><img src="icons/apps.png" alt = "apps" class = "imgMenu"/></a>
+                            <div>Apps</div>
+                        </li>
+                        <li class="elementPopUp">
+                            <a href="#"><img src="icons/documents.png" alt = "documents" class = "imgMenu"/></a>
+                            <div>Documents</div>
+                        </li>
+                    </ul>
+                    <ul class = "popUpMoreElements">
+                        <input type="checkbox" id="popUpMore">
+                        <label for="popUpMore" class="popUpBtnMore">More</label>
+
+                        <div class="popUpBodyMore">
+                            <ul class="popUpElements">
+                                <li class="elementPopUp">
+                                    <a href="#"><img src="icons/chat.png" alt = "chat" class = "imgMenu"/></a>
+                                    <div>Chat</div>
+                                </li>
+                                <li class="elementPopUp">
+                                    <a href="#"><img src="icons/classroom.png" alt = "classroom" class = "imgMenu"/></a>
+                                    <div>Classroom</div>
+                                </li>
+                                <li class="elementPopUp">
+                                    <a href="#"><img src="icons/collection.png" alt = "collection" class = "imgMenu"/></a>
+                                    <div>Collection</d>
+                                </li>
+                                <li class="elementPopUp">
+                                    <a href="#"><img src="icons/mars.png" alt = "mars" class = "imgMenu"/></a>
+                                    <div>Mars</div>
+                                </li>
+                                <li class="elementPopUp">
+                                    <a href="#"><img src="icons/blogs.png" alt = "blogs" class = "imgMenu"/></a>
+                                    <div>Blogs</div>
+                                </li>
+                                <li class="elementPopUp">
+                                    <a href="#"><img src="icons/promo.png" alt = "promo" class = "imgMenu"/></a>
+                                    <div>Promo</div>
+                                </li>
+                            </ul>
+                        </div>    
+                    </ul>
+                </div>
+            </li>
+            <li class = "elementList"><a href="#">Profile</a></li>
+        </ul>
+    </nav>
+</body>
+</html>
```

Comment:
Please, always add empty line in the end of files

Reply:
add empty line for css and html files